src/makeCalculator.js

```diff
@@ -4,7 +4,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    addOne: () => {
+      obj.result++;
+    },
```

Comment:
The `addOne` method is not required by the task and should be removed. If you want to keep a method for incrementing the result, it should be named `add` and take a number as an argument, then add that number to `result`.